docs/quickstart.rst

```diff
@@ -132,7 +151,19 @@ The machine learning workflow is the art of creating *model selection triples*,
    visualizer.fit_transform(X)
    visualizer.poof()
 
-.. image:: images/quickstart/bikeshare_rank2d.png
+.. plot::
+    :include-source: False
+    :alt: Rank2D of Bikeshare Features
```

Comment:
As you can see, only the image was replaced and by using `:include-source: False` the image is generated exactly as below while allowing us to have `code-block` examples above.

Reply:
good solution!